showcase/app/components/shw/text/index.js

```diff
@@ -136,4 +137,33 @@ export default class ShwTextIndexComponent extends Component {
 
     return classes.join(' ');
   }
+
+  @action
+  addHeadingLink(element) {
+    const innerText = element.innerText;
+    const sanitizedId = innerText
+      .trim()
+      .toLowerCase()
+      .replace(/\W/g, '-')
+      .replace(/-{2,}/g, '-')
+      .replace(/^-+|-+$/gm, '')
+      .substring(0, 64);
+
+    let uniqueId;
+    for (let i = 0; i < 100; i++) {
+      uniqueId = i > 0 ? `${sanitizedId}-${i}` : sanitizedId;
+      if (!document.getElementById(uniqueId)) {
+        break;
+      }
+    }
+    // update the heading
+    element.id = uniqueId;
+    element.classList.add('shw-page-heading-scroll-margin-top');
+    // inject an anchor element
+    const anchor = document.createElement('a');
+    anchor.href = `#${uniqueId}`;
+    anchor.className = 'shw-page-heading-link';
+    anchor.setAttribute('aria-labelledby', uniqueId);
```

Comment:
I don't think you need this. The link is already getting an accessible name from the content of the link. By the time it renders to the page it has an accessible name.

Reply:
@MelSumner can you double check (please 🙏) in the browser? 
https://hds-showcase-git-showcase-add-anchor-links-to-775174-hashicorp.vercel.app/
we do something very similar in the website, but in that case we use `aria-label` (because these are empty links, they don't wrap the text content).